apps/src/templates/tables/QuickActionsCell.jsx

```diff
@@ -1,30 +1,44 @@
 import React, {Component, PropTypes} from 'react';
+import Radium from 'radium';
 import color from "../../util/color";
 import PopUpMenu from "@cdo/apps/lib/ui/PopUpMenu";
 import styleConstants from '../../styleConstants';
 import throttle from 'lodash/debounce';
+import FontAwesome from '../FontAwesome';
+
+export const QuickActionsCellType = {
+  header: 'header',
+  body: 'body'
+};
 
 const styles = {
-  actionButton:{
-    border: '1px solid ' + color.white,
+  icon: {
     paddingLeft: 5,
     paddingRight: 5,
     paddingTop: 4,
     paddingBottom: 4,
-    borderRadius: 5,
-    color: color.lighter_gray,
-    margin: 3,
     cursor: 'pointer',
   },
+  actionButton: {
+    [QuickActionsCellType.body]: {
+      border: '1px solid ' + color.white,
+      borderRadius: 5,
+      color: color.lighter_gray,
+      margin: 3,
+    },
+    [QuickActionsCellType.header]: {
+      fontSize: 20,
+      marginTop: -3,
```

Comment:
Negative margins are not the end of the world, but can be kind of hard to be debug. Was this the only way to do this?

Reply:
It was the easiest way I could think of to do it.  The icon has a bigger font size than the text and I wanted it to look more or less vertically aligned.  Is there a better way?